### DISCUSSION TITLE
Numeric Addresses Converted to LOCALHOST and Failing to Load on LAN computers.

### DISCUSSION THREAD

**Discussion by wolfiejunk** [leokhoa/laragon#714] [Q&A]
Largon is installed and there is a Wordpress site on it, "eco1" at "192.168.1.5". 

In the browser, when I enter "192.168.1.5/eco1" it is changed to "localhost/eco1". On the computer hosting Laragon this is not a problem, but on other computers on the LAN this fails, because of course, only the hosting computer can use localhost. 

How do I prevent this from occurring, so that other computers on the LAN can access the site using the numeric address?

**Comment by wolfiejunk** [comment]
This is a really strange issue. 

Laragon was working great on Windows 10, and it's a superb product. Some time ago, I upgraded to Window 11 and Laragon started with the localhost issue. I was busy, and could do work online, so it was not 100% necessary to have Wordpress loaded into Laragon. 

Recently, because it is so very fast, I wanted to start using it again, and I began to search for a solution to the issue I posted 2 days ago. There has to be a setting or modification that would take care of this, but I can't find it. Any help would be appreciated.

Thanks,

Rich

**Comment by rescenic** [comment]
Try reinstall your Windows.

**Reply by wolfiejunk** [reply]
Thanks Rescenic. 

Late last night I found the problem. Wordpress had localhost as the database name. What was confusing and odd was that these sites always had that setting and worked fine in Laragon under Win 10. I have no explanation of why after the upgrade to Win 11 things stopped working. 

Changing the database name to the numeric IP address solved the issue.
